### ISSUE TITLE
Can not select folder endpoint

### ISSUE THREAD

**Issue by Lucstay11** [lerte/zroker#4]
Hello 
i can't share my file choose folder endpoint to share my file,can you add input folder please?

Thanks!!

**Comment by lerte** [comment]
Sorry currently use official sdk.
It does not support driver as backend mode.

**Comment by Lucstay11** [comment]
> Sorry currently use official sdk. It does not support driver as backend mode.

Yes okay, but your application does not work at all for the moment, I absolutely needed a graphical interface for zrok!

**Comment by lerte** [comment]
I don't think this app does not work.
proxy,web as backend mode work fine.
Other mode, I'm not tested.
I didn't put much effort into this.
![image](https://github.com/lerte/zroker/assets/15228321/5de68a63-1734-481f-8a5c-fcba4fe63099)

If you really need graphical interface for local tunnel.
You can try another app [Ngroker](https://ngroker.com), also written by me.

**Comment by Lucstay11** [comment]
You are very nice, but I specially turned to Zrok for the OpenSource. I don't trust Ngrok. I do not understand it's not complicated to add an option in your application to choose the folder to share ...? And can you tell me how to build my application in .deb for Linux?

**Comment by lerte** [comment]
This is project is base on [Wails](https://wails.io/).
For compile to linux  deb application.
You need:
- Linux OS
- [Go](https://go.dev) development env
- [Wails](https://wails.io/) development
- Frontend development [Node.js](https://nodejs.org/) and [pnpm](https://pnpm.io)

After that:

1. ``` git clone https://github.com/lerte/zroker.git ```
2. ```pnpm install```
3. ```wails dev```
4. ```wails build```

**Comment by Lucstay11** [comment]
I only see windows and darwin build,not Linux... And the wails docs  not specify .deb build,only binary app

**Comment by lerte** [comment]
Here is a project that also uses wails, it can be compiled into linux deb, you can refer to it.
https://github.com/tiny-craft/tiny-rdm
